<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>338</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>338</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>338</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="meeting_id_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans Condensed</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Meeting 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="summary_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>811</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Fira Code</family>
       </font>
      </property>
      <property name="text">
       <string>ðŸ“…  2025-03-25  |  ðŸ’° Total: Ksh. 12000  | Required: Ksh. 300  | Type: Standard</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="contrib_table">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>971</width>
        <height>171</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>240</number>
      </attribute>
      <column>
       <property name="text">
        <string>Full Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Method Of Payment</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date Paid</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="view_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>View The Meeting</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
